modules/openxr/doc_classes/OpenXRAPIExtension.xml

```diff
@@ -230,13 +230,27 @@
 				Registers the given extension as a composition layer provider.
 			</description>
 		</method>
+		<method name="register_frame_info_extension">
+			<return type="void" />
+			<param index="0" name="extension" type="OpenXRExtensionWrapperExtension" />
+			<description>
+				Registers the given extension as modifying frame info via the [method OpenXRExtensionWrapperExtension._set_frame_wait_info_and_get_next_pointer], [method OpenXRExtensionWrapperExtension._set_view_locate_info_and_get_next_pointer] or [method OpenXRExtensionWrapperExtension._set_frame_end_info_and_get_next_pointer] virtual methods.
+			</description>
+		</method>
 		<method name="register_projection_views_extension">
 			<return type="void" />
 			<param index="0" name="extension" type="OpenXRExtensionWrapperExtension" />
 			<description>
 				Registers the given extension as a provider of additional data structures to projections views.
 			</description>
 		</method>
+		<method name="set_custom_play_space">
+			<return type="void" />
+			<param index="0" name="space" type="const void*" />
+			<description>
+				Sets the reference space used by OpenXR to the given[url=https://registry.khronos.org/OpenXR/specs/1.0/man/html/XrSpace.html]XrSpace[/url] cast a [code]void *[/code].
```

Comment:
Is this sentence missing a comma / period?

Reply:
The formatting was definitely not ideal. I just pushed a new version - is this one easier to follow?